src/utils/clm_time_manager.F90

```diff
@@ -1274,6 +1275,29 @@ end function get_calendar
 
   !=========================================================================================
 
+  function get_doy_tomorrow(doy_today) result(doy_tomorrow)
+
+    !---------------------------------------------------------------------------------
+    ! Given a day of the year (doy_today), return the next day of the year
+
+    integer, intent(in) :: doy_today
+    integer             :: doy_tomorrow
+    character(len=*), parameter :: sub = 'clm::get_doy_tomorrow'
+
+    if ( doy_today < 1 .or. doy_today > 367 )then
```

Comment:
Why 367? Is it 367 in leap years only? If so, I suggest replacing 367 with `curr_days_per_year + 1` and explaining why `+ 1` in a comment.

Reply:
Good point. I don't know why it was EVER 367; fixed in 54e5f2470.